packages/runtime/src/widgets/Collapsible.tsx

```diff
@@ -188,14 +188,21 @@ export const Collapsible: React.FC<CollapsibleProps> = (props) => {
         },
       }}
     >
-      <Collapse
-        accordion={values?.isAccordion}
-        activeKey={activeValue}
-        expandIcon={expandIcon}
-        expandIconPosition={props.expandIconPosition}
-        items={collapsibleItems}
-        onChange={handleCollapsibleChange}
-      />
+      <div
+        onClick={(e) => e.stopPropagation()}
+        onKeyDown={(e) => e.stopPropagation()}
+        role="button"
+        tabIndex={0}
```

Comment:
These aren't necessary. If you're getting ESLint errors, let's ignore them for now.